packages/mc-html-template/html-docs/application.html

```diff
@@ -1,35 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
-    <meta name="referrer" content="no-referrer">
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, maximum-scale=1"
+    />
+    <meta name="referrer" content="no-referrer" />
 
-    <meta http-equiv="Content-Security-Policy" content="__CSP__">
+    <meta http-equiv="Content-Security-Policy" content="__CSP__" />
 
     <!-- Preconnects -->
-    <link rel="preconnect" href="__CDN_URL__">
-    <link rel="preconnect" href="__MC_API_URL__">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link rel="preconnect" href="https://app.launchdarkly.com">
-    <link rel="preconnect" href="https://clientstream.launchdarkly.com">
-    <link rel="preconnect" href="https://events.launchdarkly.com">
```

Comment:
I'm wondering about all this style changes 🤔 

I believe we should only be applying the expected inline svg changes.

Reply:
This change was made by Prettier - should I ignore Prettier for now?

Reply:
Yes, that would be great. Please 🙏